src/App.tsx

```diff
@@ -1,27 +1,61 @@
-import React from 'react';
-import './App.css';
+import React, {useState} from 'react';
+import './App.scss';
 import ErrorBoundary from './ErrorBoundary/ErrorBoundary'
+import Header from './components/Header/Header'
+import Moviecontainer from './components/Moviecontainer/Moviecontainer';
+import {Movie} from './common/types/movie'
 
 function App() {
+  const [movieList, setMovieList] = useState<Movie[]>([
+    {
+      title: 'Pulp Ficiton',
+      releaseDate: 1992,
+      type: 'crime',
+      // coverImagePath: './coverImages/pulpFiciton.jpeg'
+      coverImagePath: 'pulpFiction'
+
+    },
+    {
+      title: 'Kill Bill',
+      releaseDate: 2007,
+      type: 'comedy',
+      // coverImagePath: './coverImages/killBill.jpeg'
+      coverImagePath: 'killBill'
+    },
+    {
+      title: 'Pulp Ficiton',
+      releaseDate: 1992,
+      type: 'crime',
+      // coverImagePath: './coverImages/pulpFiciton.jpeg'
+      coverImagePath: 'pulpFiction'
+
+    },
+    {
+      title: 'Kill Bill',
+      releaseDate: 2007,
+      type: 'comedy',
+      // coverImagePath: './coverImages/killBill.jpeg'
+      coverImagePath: 'killBill'
+    },
+    {
+      title: 'Test Docu',
+      releaseDate: 2007,
+      type: 'documentary',
+      // coverImagePath: './coverImages/killBill.jpeg'
+      coverImagePath: 'killBill'
+    }
+  ])
+
   return (
     <ErrorBoundary>
       <div className="App">
-        <header className="App-header">
-          <p>
-            Edit <code>src/App.tsx</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+          <Header />
+          <Moviecontainer 
```

Comment:
Usually, it is preferred to use PascalCase in such scenarios, reading: `MovieContainer`